lib/solidus_importer/processors/product_images.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module SolidusImporter
+  module Processors
+    class ProductImages < Base
+      def call(context)
+        @data = context[:data]
+        @entity = context[:entity]
```

Comment:
Since this will necessarily be working on a product wouldn't it be more clear if the context key was `:product` instead of `:entity`? Also would suggest using `.fetch` where we require something to be there.

Reply:
> Since this will necessarily be working on a product wouldn't it be more clear if the context key was `:product` instead of `:entity`?

I'm not completely convinced about this one: `:entity` is already prepared by the Base processor so for me it's like a standard. Like: _every processor that imports something will create the entity key_.

Otherwise, I should create a key for each type, in this case, the Product processor should create `:product` in place of `:entity`? The pro here is that I could avoid the check about the entity as Product...

> Also would suggest using `.fetch` where we require something to be there.

The problem that I see about the `.fetch` method is that I should pass a context object instead of a plain hash. Another bad point for me is that if a context key is also modified I should supply an `#apply` method to save in the context (i.e. the `:data` key)

Reply:
After a call with @elia, I agree with him and will make some changes as proposed by him